### PR TITLE
Revert "Removes Belter"

### PR DESCRIPTION
**Pull request by Hopekz**
Reverts yogstation13/Yogstation#7912

Opening this because Bowl and others are sad it's gone and wanted this revert.
Original removal PR seemed rushed and undiscussed as well.

### CODE REVIEW

**Comment by swissloaf** [comment]
when does an accent become a language

**Comment by Hopekz** [comment]
> when does an accent become a language

"accent" is just an arbitrary name applied to this project.
The point of this feature is to modify your word choices.

In the case of belter it is purposefully designed so that there is a barrier to communicate. 
The argument for "Can't understand this accent" is weak as that is the point of this "accent".

It is like removing Sheltered because "can't understand these guys" when people picking sheltered make a choice to not be easily understood.

The point I am trying to make mainly is that having belter much like sheltered in the game is enjoyable for some people as it provides them with a unique experience.

**Comment by LoliconSlayer** [comment]
The argument for "Accent is just an arbitrary name" is weak when your "Arbitrary" name still remains.

The argument for "Belter is supposed to be hard to understand" is weak when it's just replacing a few choice words with a couple new ones. Anyone that actually gives a rats ass about Belter can just effectively slam their face into the keyboard and get the same effect.

The argument for "It's just the same as sheltered" is weak when Sheltered is an actual full mechanic with some tangible downsides that you chose to get when your accents is borderline half a damn mechanic that's just replacing a choice word here and there with literally zero tangible downsides whatsoever when you can just use a synonym that's not on the word list.

Either make a fucking accent that's actually a fucking accent or make it a damn language and stop beating around the proverbial bush with your roundabout reasoning.

TL;DR:
1. You call them accents, everyone else call them accents, the code calls them accents, call them whatever you want whenever we criticize Belter, they're fucking accents.
2. Your "Barrier to communicate" is poorly designed and extremely easy to do client side if someone really cares that much.
3. Sheltered actually does something unlike Belter.

**Comment by Hopekz** [comment]
> TL;DR:
> 
> 1. You call them accents, everyone else call them accents, the code calls them accents, call them whatever you want whenever we criticize Belter, they're fucking accents.

Ok. Sounds like the issue here is that we need to rename the whole project to something else.
It doesn't have to be "accents". That is literally just the name I started with and the project evolved from that point. 
Your PR didn't change the name of "accents" in general it removed the belter "accent".

> 2. Your "Barrier to communicate" is poorly designed and extremely easy to do client side if someone really cares that much.

How is this easy to do Client-side? Not everyone can consistently type in this fashion. 
I certainly can't I don't know how you can expect everyone else to be as good as you at naturally converting their thoughts to a different syntax as they type.

> 3. Sheltered actually does something unlike Belter.

Can you elaborate? Both Sheltered and Belter do something. Belter does things through the "accent" framework.

**Comment by Mr-Space** [comment]
back in my day you had to RP if you wanted to have a cool accent, zoomers today sure have it easy

**Comment by alexkar598** [comment]
Back in my days, I could understand the fucking chat because it wasnt gibberish

**Comment by Hopekz** [comment]
Not sure what I'm getting from this except I think enough people care about belter to justify keeping it in.
It's a feature not everyone will like (but then again what feature is liked by all) but because it does provide a unique experience to the people that like it; it should be kept in my opinion.

Right now there is no way to recreate the belter experience besides what LoliconSlayer said with about it being "extremely easy to do client side if someone really cares that much." 
.. so someone needs to use a macro or a script....or type out this? (I can't type like this on the fly personally so it would be for unskilled people like me)

**Comment by Ktlwjec1** [comment]
No, it's stupidly difficult to understand this to the point where I had to go on a translator to attempt to figure out what language this was before realising it was an accent. Accents are fine but ones like this are not.
